common/src/main/java/com/walmartlabs/concord/common/secret/HashAlgorithm.java

```diff
@@ -0,0 +1,38 @@
+package com.walmartlabs.concord.common.secret;
+
+/*-
+ * *****
+ * Concord
+ * -----
+ * Copyright (C) 2017 - 2022 Walmart Inc.
+ * -----
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =====
+ */
+
+import java.util.Arrays;
+
+public enum HashAlgorithm {
+    MD5("md5"),
```

Comment:
Let's call it `LEGACY_MD5` and maybe annotate it `@Deprecated` on it too?
We probably want to discourage any further use of the old algo.

Reply:
Updated with LEGACY_MD5 and added @deprecated tag